app/src/main/java/com/example/nicocalculator/MainActivity.kt

```diff
@@ -2,11 +2,20 @@ package com.example.nicocalculator
 
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        text_results.text = "Result"
```

Comment:
Please, move the hardcoded strings to a constant in the class.

Reply:
Done